Peer.Domain/Filters/IPropertySelector.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Peer.Domain.Filters
+{
+    public interface IPropertySelector
+    {
+        Func<PullRequest, IComparable> Selector { get; }
```

Comment:
Could be declared as a normal method member? I think anything needing a `Func<A, B>` can accept a reference to `x.Foo` where `x` is an object with a member `B Foo(A a)`.

Reply:
I don't recall the usage so maybe this makes more sense semantically. It looked strange at first and got me thinking about whether there was a functional difference. I might not suggest the change but I'm curious about the equivalence now either way.